PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -252,10 +252,201 @@ struct HfTrackIndexSkimCreatorTagSelCollisions {
+              trackTgl.push_back(trackParCov.getTgl());
+              trackSigned1Pt.push_back(trackParCov.getQ2Pt());
+              trackDcaXY.push_back(dcaInfoCov.getY());
+              trackDcaZ.push_back(dcaInfoCov.getZ());
+              trackSigmaY.push_back(std::sqrt(trackParCov.getSigmaY2()));
+              trackSigmaZ.push_back(std::sqrt(trackParCov.getSigmaZ2()));
+              trackSigmaSnp.push_back(std::sqrt(trackParCov.getSigmaSnp2()));
+              trackSigmaTgl.push_back(std::sqrt(trackParCov.getSigmaTgl2()));
+              trackSigma1Pt.push_back(std::sqrt(trackParCov.getSigma1Pt2()));
+            }
+          }
+        }
+      }
+    } /// end for loop ambiguous tracks
+
+    /// fill the table with tracks compatible in time with bc
+    for (auto& bc : bcWithTimeStamps) {
+
+      std::vector<int> collIDs = {};
+      for (auto& coll : collisions) {
+        /// count how many coll. have the current bc as most probable
+        if (bc.globalBC() == coll.bc_as<aod::BCsWithTimestamps>().globalBC()) {
+          collIDs.push_back(coll.globalIndex());
+        }
+      } /// end loop on collisiuons
+
+      /// we do not want bcs with only 1 coll
+      if (collIDs.size() < 2) {
+        continue;
+      }
+
+      for (int& collID : collIDs) {
+        /// tracks for the current collision
+        const auto& tracksColl = tracks.sliceBy(perRecoCollision, collID);
+
+        for (auto& track : tracksColl) {
+          /// select the tracks according to: 1) track selection; 2) is PV contributor or not
+          if ((!useIsGlobalTrackWoDCA || (useIsGlobalTrackWoDCA && track.isGlobalTrackWoDCA())) && track.isPVContributor()) {
+            // track selection done wrt the "default" vertex (~ equivalent to IU point)
+            for (int& collIDother : collIDs) {
+              if (collIDother == collID) { /// this is already considered in the standard track table
+                continue;
+              }
+              auto collision = collisions.rawIteratorAt(collIDother);
+
+              /// At this point, wthe remaining collisions are by definition compatible in time
+              initCCDB(bc, runNumber, ccdb, ccdbPathGrpMag, lut, false);
+
+              // let's propagate track to collision
+              dcaInfoCov.set(999, 999, 999, 999, 999);
+              auto trackParCov = getTrackParCov(track);
+              vtx.setPos({collision.posX(), collision.posY(), collision.posZ()});
+              vtx.setCov(collision.covXX(), collision.covXY(), collision.covYY(), collision.covXZ(), collision.covYZ(), collision.covZZ());
+              o2::base::Propagator::Instance()->propagateToDCABxByBz(vtx, trackParCov, 2.f, matCorr, &dcaInfoCov);
+
+              /// Fill the table with this track propagated to the new collisions
+              collIndices.push_back(collision.globalIndex());
+              trackIndices.push_back(track.globalIndex());
+              trackTypes.push_back(BIT(hf_amb_tracks::Ambiguous));
```

Comment:
@fgrosa here probably we wrongly changed the flag, it should be `hf_amb_tracks::PVContributor`